docs/releases.rst

```diff
@@ -20,6 +20,10 @@ environment files are found.
 The ``tmt try`` command now supports the new
 :ref:`/stories/cli/try/option/arch` option.
 
+Add support for ``--keep`` params for the ``clean_guests`` subcommand.
```

Comment:
This should now be under 1.40

Reply:
Thanks for your review: https://github.com/teemtee/tmt/pull/3182/commits/50cf34da8accf67d1d3de0a96b02cd88ea43fc69